shared/.babelrc

```diff
@@ -1,27 +1,18 @@
 {
-  // React native doesn't support env so by default we use react-native and in electron we specify it
-  // See: https://github.com/facebook/react-native/issues/8723
-  "presets": [
-    "babel-preset-react-native"
-  ],
-  "plugins": [
-    [
-      "babel-plugin-transform-builtin-extend",
-      {
-        "globals": [
-          "Error"
-        ]
-      }
-    ]
-  ],
   "env": {
+    "development": {
```

Comment:
leaving this in the root (old way) doesn't seem to work anymore. leaving it in development does seem to work